tests/neo-vm.Tests/Tests/OpCodes/Arrays/PICKITEM.json

```diff
@@ -567,7 +567,7 @@
                     "value": 4
                   },
                   {
-                    "type": "byteArray",
+                    "type": "String",
                     "value": "0x0102"
```

Comment:
Type is `String`, but our value is hex string?

Reply:
Strings in VM it's a fixed byte array

Reply:
Maybe `String` is not a best name, how about `ByteString`, `Bytes`, `ImmutableBytes`, or `ByteArray`? @vncoelho Could you give some advice?

Reply:
I  think we can keep `ByteArray`, we need to create NEP for Neo3-VM standard to introduce those basic types, like python [PEP 358: bytes](https://www.python.org/dev/peps/pep-0358/).

Reply:
`ByteString` is good. I will rename to it.